src/Tizen.NUI.Components/Controls/Scrollbar.cs

```diff
@@ -475,6 +527,43 @@ private void UpdateTrackPadding(Extents padding)
             thumbView.Size = calculator.CalculateThumbSize(ThumbThickness, trackView.Size);
             thumbView.Position = calculator.CalculateThumbPaddingPosition(trackView.Size, thumbView.Size, thumbView.Position, trackPadding);
         }
+        private void UpdateThumbColor(Color color)
+        {
+            thumbColor = color;
+            if (String.IsNullOrEmpty(thumbView.ResourceUrl))
+            {
+                thumbView.Color = color;
```

Comment:
Does it work? The property `Color` is a mixed color value that will be multiplied with other visuals color in a View. So I think when there is no resource url, it can be just a backgroundColor.

Reply:
This is typo after change build error.
the condition must be !String.IsNullOrEmpty, so if thumeView resourceUrl is exist, set color on the view.